generic/SpatialConvolutionLocal.c

```diff
@@ -0,0 +1,414 @@
+#ifndef TH_GENERIC_FILE
+#define TH_GENERIC_FILE "generic/SpatialConvolutionLocal.c"
+#else
+
+#ifdef _WIN32
+# include <windows.h>
+#endif
+
+
+
+/* note: due to write issues, this one cannot be parallelized as well as unfolded_copy */
+static void nn_(spatialconvolutionlocal_unfolded_acc)(THTensor *finput, THTensor *input,
```

Comment:
I thought about it before but the problem is that, previously each file in `nn/generic` corresponds to a module. If I add something like `unfold.c`, `unfold.h`, it may looks a little inconsistent.  
Anyway, I've just made a commit with `unfolded_copy` and `unfolded_acc` in a separate file. You can see if it's better.